connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -122,6 +124,18 @@ public class WorkerConfig extends AbstractConfig {
             + "by the worker's scanner before config providers are initialized and used to "
             + "replace variables.";
 
+    public static final String PLUGIN_DISCOVERY_CONFIG = "plugin.discovery";
+    protected static final String PLUGIN_DISCOVERY_DOC = "Method to use to discover plugins provided in the "
+            + "plugin.path configuration. This can be one of multiple values with the following meanings:\n"
+            + "* ONLY_SCAN: Discover plugins only by reflection. "
+            + "Plugins which are not discoverable by ServiceLoader will not impact worker startup.\n"
+            + "* HYBRID_WARN: Discover plugins reflectively and by ServiceLoader. "
+            + "Plugins which are not discoverable by ServiceLoader will print warnings during worker startup.\n"
+            + "* HYBRID_FAIL: Discover plugins reflectively and by ServiceLoader."
```

Comment:
```suggestion
            + "* HYBRID_FAIL: Discover plugins reflectively and by ServiceLoader. "
```

Can't wait for Java 15 text blocks 🙏